src/qiskit_qec/utils/stim_tools.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name, disable=no-name-in-module
+
+"""Tools to use functionality from Stim."""
+from typing import List, Dict
+from math import log
+from stim import Circuit as StimCircuit
+from stim import DetectorErrorModel as StimDetectorErrorModel
+from stim import DemInstruction as StimDemInstruction
+from stim import DemTarget as StimDemTarget
+
+import numpy as np
+import rustworkx as rx
+
+
+from qiskit import QuantumCircuit
+from qiskit_aer.noise.errors.quantum_error import QuantumChannelInstruction
+from qiskit_aer.noise import pauli_error
+from qiskit_qec.utils.decoding_graph_attributes import DecodingGraphNode, DecodingGraphEdge
+
+
+def get_stim_circuits(circuit_dict):
```

Comment:
Can you move the typing information from the args docstring into the args using typing? Something like Dict[Int, QuantumCircuit]

Reply:
Done